tools/disassemble_sequence.py

```diff
@@ -0,0 +1,792 @@
+            continue
+            
+        if ".define" in line and current_mode != "":
+            sym, index = re.match(r"\.define ([A-Z_0-9]+)\s+(\d+)", line).groups()
+            if current_mode == "instrument":
+                inst_syms[int(index)] = sym
+            elif current_mode == "drum":
+                drum_syms[int(index)] = sym
+            elif current_mode == "effect":
+                effect_syms[int(index)] = sym
+
+    return (inst_syms, drum_syms, effect_syms)
+
+def get_sequence_symbols(seq_inc):
+    pitches = {}
+    fonts = {}
+
+    for line in seq_inc.readlines():
+        if "PITCH" in line:
+            sym, id = re.match(r"\.define ([A-Z_0-9]+)\s+(\d+)", line).groups()
+            pitches[int(id)] = sym
+        if "SOUNDFONT" in line:
+            sym, id = re.match(r"\.define ([A-Z_0-9]+)\s+(\d+)", line).groups()
+            fonts[int(id)] = sym
+
+    return (pitches, fonts)
+
+control_flow_commands = {
+    0xFF: ['end'],
+    0xFE: ['delay1'],
+    0xFD: ['delay', 'var'],
+    0xFC: ['call', 'addr'],
+    0xFB: ['jump', 'addr'],
+    0xFA: ['beqz', 'addr'],
+    0xF9: ['bltz', 'addr'],
+    0xF8: ['loop', 'u8'],
+    0xF7: ['loopend'],
+    0xF6: ['break'],
+    0xF5: ['bgez', 'addr'],
+    0xF4: ['rjump', 'reladdr8'],
+    0xF3: ['rbeqz', 'reladdr8'],
+    0xF2: ['rbltz', 'reladdr8'],
+}
+
+commands = {}
+commands['seq'] = join(control_flow_commands, {
+    # non-arg commands
+    0xF1: ['allocnotelist', 'u8'],
+    0xF0: ['freenotelist'],
+    0xDF: ['transpose', 's8'],
+    0xDE: ['rtranspose', 's8'],
+    0xDD: ['tempo', 'u8'],
+    0xDC: ['tempochg', 's8'],
+    0xDB: ['vol', 'u8'],
+    0xDA: ['volmode', 's8'],
+    0xD7: ['initchan', 'hex16'],
+    0xD6: ['freechan', 'hex16'],
+    0xD5: ['mutescale', 's8'],
+    0xD4: ['mute'],
+    0xD3: ['mutebhv', 'hex8'],
+    0xD2: ['ldshortvelarr', 'addr'],
+    0xD1: ['ldshortgatearr', 'addr'],
+    0xD0: ['notealloc', 'u8'],
+    0xCE: ['rand', 'u8'],
+    0xCD: ['dyncall', 'addr'],
+    0xCC: ['ldi', 'u8'],
+    0xC9: ['and', 'u8'],
+    0xC8: ['sub', 'u8'],
+    0xC7: ['stseq', 'u8', 'addr'],
+    0xC6: ['stop'],
+    0xC5: ['scriptctr', 'u16'],
+    0xC4: ['runseq', 'u8', 'u8'],
+    # arg commands
+    0x00: ['testchan', 'bits:4'],
+    0x40: ['stopchan', 'bits:4'],
+    0x50: ['subio', 'bits:4'],
+    0x60: ['ldres', 'bits:4', 'u8', 'u8'],
+    0x70: ['stio', 'bits:4'],
+    0x80: ['ldio', 'bits:4'],
+    0x90: ['ldchan', 'bits:4', 'addr'],
+    0xA0: ['rldchan', 'bits:4', 'reladdr16'],
+    0xB0: ['ldseq', 'bits:4', 'u8', 'addr'],
+})
+
+commands['chan'] = join(control_flow_commands, {
+    # non-arg commands
+    0xF1: ['allocnotelist', 'u8'],
+    0xF0: ['freenotelist'],
+    0xEE: ['bendfine', 's8'],
+    0xED: ['gain', 'u8'],
+    0xEB: ['fontinstr'],
+    0xEA: ['stop'],
+    0xE9: ['notepri', 'u8'],
+    0xE8: ['params', 'u8', 'u8', 'u8', 's8', 's8', 'u8', 'u8', 'u8'],
+    0xE7: ['ldparams', 'addr'],
+    0xE6: ['samplbook', 'addr'],
```

Comment:
```suggestion
    0xE6: ['samplebook', 'addr'],
```
I think this can afford the extra e

Reply:
Agreed, applied in #1509.